src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,16 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int weight;
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. You should use getters and setters to access the `weight` field instead of exposing it directly.